client/www/scripts/modules/pm/pm.services.js

```diff
@@ -34,7 +34,7 @@ PM.service('PMAppService', [
           return response;
         })
         .catch(function(error) {
-          $log.error('bad deploy local app: ' + error.message);
+          $log.error('bad deploy local app: ' + JSON.stringify(error));
```

Comment:
Error should have a `message` field. Is that missing? Is it a real Error object?